docs/patterns/components/Pagination/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import Demo from '../../../Demo';
+import { Pagination } from '@deque/cauldron-react/';
+import { children, className } from '../../../props';
+
+const PaginationDemo = () => {
+  const totalItems = 111;
+  const itemsPerPage = 25;
+  const [currentPage, setCurrentPage] = useState(3);
+  const itemStart = currentPage * itemsPerPage - itemsPerPage + 1;
+  const itemEnd = Math.min(itemStart + itemsPerPage - 1, totalItems);
+
+  const onNextPageClick = () => setCurrentPage(currentPage + 1);
+  const onFirstPageClick = () => setCurrentPage(1);
+  const onPreviousPageClick = () => setCurrentPage(currentPage - 1);
+  const onLastPageClick = () => setCurrentPage(5);
+
+  return (
+    <div>
+      <Demo
+        component={Pagination}
+        states={[
+          { totalItems: 15 },
+          // TODO: don't render this when in light mode?
```

Comment:
I'll need to get clarification on this. The designs only have this additional variant in dark mode

Reply:
Do not include the additional variant at all, per @awpearlm